docs/decisions/0002-lti-1p3-variables.rst

```diff
@@ -0,0 +1,161 @@
+LTI 1.3 variable definition
+---------------------------
+
+Status
+======
+
+Provisional
+
+Context
+=======
+
+The LTI 1.3 specification defines multiple variables related to course, context, problem, student id and PII.
+
+The goal of this ADR is to provide well defined mappings from variables available in the platform and
+variables required by LTI 1.3 launches.
+
+Decisions
+=========
+
+The following section go over required and optional variables in each supported claim in the LTI 1.3 implementation.
+This ADR should be updated when new claims are added.
+
+Deployment ID
+~~~~~~~~~~~~~
+
+.. list-table::
+   :widths: auto
+   :header-rows: 1
+
+   * - LTI 1.3 variable name
+     - Open edX
+     - Reasoning
+   * - https://purl.imsglobal.org/spec/lti/claim/deployment_id
+     - "1" (fixed value)
+     - We're using single-tenant deployments, were each key is used for a single XBlock. Even when moving up to the course level,
+       we'll can keep this setup since multi-tenancy will only be required for instance wide configuration.
+
+See http://www.imsglobal.org/spec/lti/v1p3/#single-tenant-tool-registered-and-deployed-once for more details.
+
+Roles
+~~~~~
+The roles claim takes in a list of LTI 1.3 compliant roles from http://www.imsglobal.org/spec/lti/v1p3/#role-vocabularies.
+The claim name is: https://purl.imsglobal.org/spec/lti/claim/roles.
+
+The mappings from edX platform are shown in the table below:
+
+.. list-table::
+   :widths: auto
+   :header-rows: 1
+
+   * - Open edX role
+     - LTI 1.3 Roles included
+     - Reasoning
+   * - guest
+     - Empty
+     - Guests are users not logged in, they shoudn't be able to access LTI content.
+   * - student
+     - 'http://purl.imsglobal.org/vocab/lis/v2/institution/person#Student'
+     - Students only have permission to view and interact with the LTI content.
+   * - instructor
+     - 'http://purl.imsglobal.org/vocab/lis/v2/institution/person#Instructor'
```

Comment:
Why Instructor instead of Staff?

Reply:
We don't want to give instructors staff permissions, no?
In my understanding, `instructors` don't have the same level of access that `staff` (course creators) do.

Is this assumption wrong?

Reply:
We need to remove Student from the instructor and staff lists.

Reply:
@nedbat Done.